tests/integration_tests/core_tests.py

```diff
@@ -117,15 +117,6 @@ def test_dashboard_endpoint(self):
     @pytest.mark.usefixtures("load_birth_names_dashboard_with_slices")
     def test_slice_endpoint(self):
         self.login(username="admin")
-        slc = self.get_slice("Girls", db.session)
-        resp = self.get_resp("/superset/slice/{}/".format(slc.id))
-        assert "Original value" in resp
-        assert "List Roles" in resp
-
-        # Testing overrides
-        resp = self.get_resp("/superset/slice/{}/?standalone=true".format(slc.id))
-        assert '<div class="navbar' not in resp
```

Comment:
if this is false now, what's the current value? Do we still have a standalone url without the nav?

Reply:
I believe that now that Explore is in the SPA, the response doc just contains the SPA wrapper.  I was assuming that `get_resp` doesn't download/execute the JS bundle, so I think these tests were only still passing by accident:
- `"Original value"` and `"List Roles"` are contained in the embedded bootstrap data but I don't think that bootstrap data is used by Explore anymore (it uses API endpoints now)
- `<div class="navbar'` isn't present in either standalone or non-standalone mode anymore

So, it seemed like these tests don't test anything meaningful anymore, with the exception of the 404 test below.  I looked around for examples of backend tests for Dashboard, but didn't see much and assumed they had also been removed when Dashboard was brought into the SPA.